WordPressAuthenticator/Model/LoginFields.swift

```diff
@@ -36,7 +36,22 @@ public class LoginFields: NSObject {
     /// the signup flow.
     @objc public var emailAddress = ""
 
-    @objc public var meta = LoginFieldsMeta()
+    var meta = LoginFieldsMeta()
+
+    @objc public var userIsDotCom: Bool {
+        get { meta.userIsDotCom }
+        set { meta.userIsDotCom = newValue }
+    }
+
+    @objc public var requiredMultifactor: Bool {
+        meta.requiredMultifactor
+    }
+
+    @objc public var xmlrpcURL: NSURL? {
+        get { meta.xmlrpcURL }
+        set { meta.xmlrpcURL = newValue }
+    }
```

Comment:
I'm guessing the reason these properties are public is they are used by the apps?

Reply:
Correct. At least, I think that's why I set them like that, but I shall double check. Thank you for asking this 🙌

Reply:
Actually, here's why the were `public`: They are required to be so that they can be seen in `LoginFacade`:

<img width="300" alt="image" src="https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/assets/1218433/ca5f2916-9073-4213-b017-40991a5313fe">

Reply:
Interesting... I didn't know that you have to mark these property as public for them to be available to `LoginFacade.m` which sits in the same module.